warp10/src/main/sh/warp10.sh

```diff
@@ -237,15 +237,7 @@ distConf() {
   postInit
 }
 
-leveldbConf() {
-  echo "Initializing Warp 10 standalone configuration"
-  init
-
-  echo "
-backend = leveldb" >>"${WARP10_CONFIG_DIR}/99-init.conf"
-  mv "${WARP10_CONFIG_DIR}/10-fdb.conf" "${WARP10_CONFIG_DIR}/10-fdb.conf.DISABLE"
-  getConfigFiles
-
+leveldbInit() {
```

Comment:
This function does not create the `snapshots` directory, it should

Reply:
snapshots are created just before java WarpInit. Should be OK.